src/common/event/event-bus.ts

```
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-unsafe-function-type */
+import { Injectable } from '@nestjs/common';
+import { DomainEvent } from './domain-event';
+
+@Injectable()
+export class EventBus {
+  private handlers: Map<string, Function[]> = new Map();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)